### ISSUE TITLE
Failed to build on MacOs

### ISSUE THREAD

**Issue by alan-baker** [google/graphicsfuzz#87]
Ran into the following error running `mvn package`:
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-checkstyle-plugin:2.17:check (1-check-style) on project parent-checkstyle: Failed during checkstyle configuration: Cannot set property 'cacheFile' to '<root path>/graphicsfuzz/parent-checkstyle/target/checkstyle-cache.txt' in module Checker: InvocationTargetException: javax/xml/bind/DatatypeConverter: javax.xml.bind.DatatypeConverter -> [Help 1]
```

This occurred on my Macbook.

**Comment by paulthomson** [comment]
How strange. A workaround for now is to add `-Dcheckstyle.skip`. E.g.

`mvn package -Dcheckstyle.skip`

**Comment by paulthomson** [comment]
This might be fixed by #117, but I can't reproduce easily right now. We will add more CI configurations in #115. I will leave this open for now. Feel free to retry with the latest master.

**Comment by paulthomson** [comment]
Closing as I am fairly sure it is fixed and I couldn't reproduce.
